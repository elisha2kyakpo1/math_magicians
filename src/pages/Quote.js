function Quote() {
  return (
    <div>
      <p className="quote">
        Mathematics is not about numbers,
        equations, computations or algorithms: it is about understanding -William paul thurston
      </p>
    </div>
  );
}

export default Quote;
