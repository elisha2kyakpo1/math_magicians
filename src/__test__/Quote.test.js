import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../pages/Quote';

test('Should render correctly', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
