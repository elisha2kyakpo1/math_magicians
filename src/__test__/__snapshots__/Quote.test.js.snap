// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<div>
  <p
    className="quote"
  >
    Mathematics is not about numbers, equations, computations or algorithms: it is about understanding -William paul thurston
  </p>
</div>
`;
